shared/fixes/bash/ensure_logrotate_activated.sh

```diff
@@ -0,0 +1,7 @@
+# platform = multi_platform_rhel
+
+# daily rotation is configured
+grep -q "^daily$" /etc/logrotate.conf || echo "daily" >> /etc/logrotate.conf
+
+# remove any line configuring weekly, monthly or yearly rotation
+sed -i -r "/^(weekly|monthly|yearly)$/d" /etc/logrotate.conf
```

Comment:
this is not adding logrotate to crontab

Reply:
Hmm, so just configuring `logrotate.conf` is not enough?

Reply:
I don't know but I thought you have to call `/sbin/logrotate` periodically for the config file to even be used.

Reply:
It makes sense, updated everything to account for `logrotate.conf` and `cron.daily`.

Reply:
Wait, did you verify that? Because I am not sure about what I said.

Reply:
Yes, `man logrotate` states the following
> Normally, logrotate is run as a daily cron job.

Reply:
Also from man page:
> hourly       Log files are rotated every hour. Note that usually logrotate is configured to be run by cron daily. You have to change this configuration and run logrotate hourly to be able to really rotate logs hourly.

Configuring frequency of logrotation in `logrotate.conf` doesn't account for frequency of `logrotate` runs. The cron job also needs to be configured.